src/content/docs/en/guides/migrate-to-astro/from-docusaurus.mdx

```diff
@@ -16,7 +16,7 @@ import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
 
 Docusaurus and Astro share some similarities that will help you migrate your project:
 
-- Both Astro and Docusaurus are modern, JavaScript-based (Jamstack) site builders intended for [content-focused websites](/en/concepts/why-astro/#content-focused), like documentation sites. 
+- Both Astro and Docusaurus are modern, JavaScript-based (Jamstack) site builders intended for [content-heavy websites](/en/concepts/why-astro/#content-driven), like documentation sites. 
```

Comment:
Any reason we're going for `content-heavy` here instead of the new term `content-driven` like on the Why Astro page? I am not sure about making two terms for what apparently is, the same thing. 😅

Reply:
Yeah, because I kind of felt like in this one case, for documentation sites, they really are content-**heavy**, and on the migrate from docusuarus page felt more like meeting the reader where they might be at vs using our marketing phrase.

If this is terrible, I'm open to changing it!

Reply:
Updating them all to match!